### PR TITLE
Pascal's_Triangle

### PR DESCRIPTION
**Pull request by somya-kapoor**
#731

### CODE REVIEW

File: Pascal's_Triangle/Pascal's_Triangle.c [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      f *= n;
+   return f;
+}
+
+int main() 
+{
+   int n, i, j;
+   printf("Enter the numbers of rows ");
+   scanf("%d",&n);
+   for(i = 0; i < n; i++)
```
Comment (ayushin78) [inline Pascal's_Triangle/Pascal's_Triangle.c]: Having whitespace between logical blocks would enhance the readability of the code.

File: Pascal's_Triangle/Pascal's_Triangle.c [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+
+int factorial(int n)
+{
+   int f;
```
Comment (ayushin78) [inline Pascal's_Triangle/Pascal's_Triangle.c]: Avoid using single-letter variable names.

Comment (somya-kapoor) [reply Pascal's_Triangle/Pascal's_Triangle.c]: @ayushin78  Okay

**Comment by somya-kapoor** [comment]
Hey @ayushi78 !
Okay

**Comment by ayushin78** [comment]
LGTM. Thanks @somya-kapoor. 
Going ahead and merging it. :)

**Pull request merged**